packages/markdoc/package.json

```diff
@@ -17,12 +17,12 @@
     "./components": "./components.ts"
   },
   "devDependencies": {
-    "@astrojs/markdoc": "^0.11.5",
+    "@astrojs/markdoc": "^0.12.1",
     "@astrojs/starlight": "workspace:*",
-    "vitest": "^1.6.0"
+    "vitest": "2.1.6"
```

Comment:
Decided to pin Vitest version to [`2.1.6`](https://github.com/vitest-dev/vitest/releases/tag/v2.1.6) which introduced support for Vite 6. It was reverted in [`2.1.7`](https://github.com/vitest-dev/vitest/releases/tag/v2.1.7) and I was hoping this would be a quick enough fix but it looks like it may take a bit longer. As we're not affected by the issue (mostly `resolve.conditions` related), and considering support was added and removed in patch versions, I think it's better to pin to that version for now. I'll keep an eye on it, update and unpin when it's resolved.